access_log/access_log_record.go

```diff
@@ -69,6 +71,10 @@ func (r *AccessLogRecord) makeRecord() *bytes.Buffer {
 		fmt.Fprintf(b, `app_id:%s`, r.RouteEndpoint.ApplicationId)
 	}
 
+	if r.ExtraHeadersToLog != nil {
```

Comment:
Please check for empty slice.  We do not want to log an space in case ExtraHeadersToLog is an empty slice, which is the default value.

https://github.com/cloudfoundry/cf-release/pull/773